src/client/components/ContactForm/ContactForm.component.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './ContactForm.styles.css';
+
+export default function ContactForm({user}) {
+    return (
+        <div className="infoBox">
+            <div className="wrapper">
+                <form className="form">
+                    <label className="title">CONTACT:</label>
+                    <label className="fullName">Full-name:  {
+                        user.fullName
+                    }</label>
```

Comment:
```suggestion
                    <label className="fullName">Full-name:  {fullName}</label>
```

Reply:
Also check that fullName is not null.

Reply:
Made changes according to the review comments